docs/user-manuals/multi-hierarchy-elastic-quota-management.md

```diff
@@ -0,0 +1,519 @@
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+  restartPolicy: Always
+```
+
+```bash
+$ kubectl apply -f pod-example.yaml
+  pod/pod-example created
+```
+
+3.Verify `quota-example` has changed.
+```bash
+$ kubectl get eqs -n default quota-example -o yaml
+```
+```yaml
+kind: ElasticQuota
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"scheduling.sigs.k8s.io/v1alpha1","kind":"ElasticQuota","metadata":{"annotations":{},"labels":{"quota.scheduling.koordinator.sh/is-parent":"false","quota.scheduling.koordinator.sh/parent":""},"name":"quota-example","namespace":"default"},"spec":{"max":{"cpu":40,"memory":"40Gi","nvidia.com/gpu":2},"min":{"cpu":10,"memory":"20Mi","nvidia.com/gpu":1}}}
+    quota.scheduling.koordinator.sh/request: '{"cpu":"40m","memory":"40Mi"}'
+    quota.scheduling.koordinator.sh/runtime: '{"cpu":"40m","memory":"40Mi"}'
+    quota.scheduling.koordinator.sh/shared-weight: '{"cpu":"40","memory":"40Gi"}'
+  creationTimestamp: "2022-10-08T09:26:38Z"
+  generation: 2
+  labels:
+    quota.scheduling.koordinator.sh/is-parent: "false"
+    quota.scheduling.koordinator.sh/parent: root
+    manager: koord-scheduler
+    operation: Update
+    time: "2022-10-08T09:26:50Z"
+  name: quota-example
+  namespace: default
+  resourceVersion: "39012008"
+  selfLink: /apis/scheduling.sigs.k8s.io/v1alpha1/namespaces/default/elasticquotas/quota-example
+  uid: 936b1a5d-8404-41d1-87fa-fd26eb73c78c
+spec:
+  max:
+    cpu: "40"
+    memory: 40Gi
+  min:
+    cpu: "10"
+    memory: 20Mi
+status:
+  used:
+    cpu: 40m
+    memory: 40Mi
+```
+
+4.Verify `quota-example` debug api.
+```bash
+$ kubectl -n koordinator-system get lease koord-scheduler --no-headers | awk '{print $2}' | cut -d'_' -f1 | xargs -I {} kubectl -n koordinator-system get pod {} -o wide --no-headers | awk '{print $6}'
+$ curl $schedulerip::10251/apis/v1/plugins/ElasticQuota/quota/quota-example
+```
+
+```json
+{
+    "allowLentResource": true,
+    "autoScaleMin": {
+        "cpu": "10",
+        "memory": "20Mi",
+    },
+    "isParent": false,
+    "max": {
+        "cpu": "40",
+        "memory": "40Gi",
+    },
+    "min": {
+        "cpu": "10",
+        "memory": "20Mi",
+    },
+    "name": "quota-example",
+    "parentName": "root",
+    "podCache": {
+        "pod-example": {
+            "isAssigned": true,
+            "resource": {
+                "cpu": "40m",
+                "memory": "40Mi"
+            }
+        }
+    },
+    "request": {
+        "cpu": "40m",
+        "memory": "40Mi"
+    },
+    "runtime": {
+        "cpu": "40m",
+        "memory": "41943040",
+    },
+    "runtimeVersion": 39,
+    "sharedWeight": {
+        "cpu": "40",
+        "memory": "40Gi",
+    },
+    "used": {
+        "cpu": "40m",
+        "memory": "40Mi"
+    }
+}
+```
+The main different with yaml is that we can find all quota's pods and its status in `podCache`.
```

Comment:
Consider the example of adding one or more namespace Pods to a Quota?

Reply:
fixed